Orange/regression/linear.py

```diff
@@ -27,19 +30,50 @@ def score(self, data):
 
 class LinearRegressionLearner(SklLearner, _FeatureScorerMixin):
     __wraps__ = skl_linear_model.LinearRegression
+    __penalty__ = None
 
     # Arguments are needed for signatures, pylint: disable=unused-argument
     def __init__(self, preprocessors=None, fit_intercept=True):
         super().__init__(preprocessors=preprocessors)
         self.params = vars()
 
+    def _initialize_wrapped(self, X=None, Y=None):
+        if isinstance(X, da.Array) or isinstance(Y, da.Array):
+            try:
+                import dask_ml.linear_model
+
+                params = self.params.copy()
+                params["solver"] = "gradient_descent"
+                params["penalty"] = self.__penalty__
+                if self.__penalty__ is not None:
+                    params["solver"] = "admm"
+                    params["C"] = 1 / params.pop("alpha")
+                    params["max_iter"] = params["max_iter"] or 100
+                    for key in ["copy_X", "precompute", "positive"]:
+                        params.pop(key, None)
+                if self.__penalty__ == "elasticnet":
+                    from dask_glm.regularizers import ElasticNet
+                    params["penalty"] = ElasticNet(weight=params.pop("l1_ratio"))
+
+                return dask_ml.linear_model.LinearRegression(**params)
+            except ImportError:
+                warnings.warn("dask_ml is not installed, using sklearn instead.")
+        return self.__wraps__(**self.params)
+
     def fit(self, X, Y, W=None):
-        model = super().fit(X, Y, W)
-        return LinearModel(model.skl_model)
+        clf = self._initialize_wrapped(X, Y)
+        Y = Y.reshape(-1)
```

Comment:
If someone else ensures that this class does not work with multiple targets then reshaping to (-1) is fine. Otherwise, we'd rather select the first column and then (if needed) reshape it.